app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<section class="user-container">
+  <card class="user-item">
+    <img src='<%= @user.photo %>'></img>
+    <div class="user-data">
+    <h2><%= @user.name %></h2><br>
+    <h4 class="custom-card-t">Number of posts: <%= @user.posts_counter %> </h4>
+    </div>
+  </card>
+  <div class="posts-container">
+    <% @posts.each do |post| %>
+      <div class="post-comment-box">
+        <card class="post-item">
+          <div class="post-data">
+            <h2>Post <%= post.id %></h2>
+            <p> <%= post.text %> </p>
+            <div class="counter-data">
+              <h5 class="custom-card-p">Comments: <%= post.comments_counter %> </h5>
+              <h5 class="custom-card-p">Likes: <%= post.likes_counter %> </h5>
+            </div>
+          </div>
+        </card>
+        <div class="comments-box">
+          <% if post.comments.empty? %>
+            <p> There are no comments for this post </p>
+          <% end %>
+          <% post.comments.each do |comment| %>
+            <p> <%= comment.author.name %>: <%= comment.text %> </p>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</section>
```

Comment:
- [ ] It would be nice if you make your app more functional by adding links here to your post_show page because I cannot go to your post_show page without changing the URL 😨 Kindly make each post a link so whenever a post is clicked then it takes us to the post_show page with respective user id and post id 👍 or if you wanna do it in a different way then you can do it but you should make the user able to access the post_show page 😊